src/main/java/bio/terra/workspace/service/workspace/flight/DeleteWorkspaceAuthzStep.java

```diff
@@ -29,21 +27,10 @@ public StepResult doStep(FlightContext flightContext) throws RetryException {
     FlightMap inputMap = flightContext.getInputParameters();
     UUID workspaceID =
         UUID.fromString(inputMap.get(WorkspaceFlightMapKeys.WORKSPACE_ID, String.class));
-    try {
-      samService.deleteWorkspace(userReq.getRequiredToken(), workspaceID);
-    } catch (SamApiException e) {
-      // Stairway steps may run multiple times, so this may already have been deleted.
-      // For all other errors we should always retry because there's no way to undo a Sam delete.
-      if (e.getApiExceptionStatus() != HttpStatus.NOT_FOUND.value()) {
-        logger.warn(
-            String.format(
-                "Signaling retry for unexpected Sam error: code (%d) error: %s",
-                e.getApiExceptionStatus(), e.toString()),
-            e);
-        return new StepResult(StepStatus.STEP_RESULT_FAILURE_RETRY, e);
-      }
-    }
-    return StepResult.getStepResultSuccess();
+    // Sam Service will take care of retrying any retry-able errors. Failures will be bubbled up
+    // as derivatives of ErrorReportException.
```

Comment:
nit: funny this step but only this step gets a comment. Maybe just remove?

Reply:
Done